src/app.js

```diff
@@ -7,6 +7,19 @@ export function app(app) {
   var node
   var element
 
+  function hydrate(elm) {
+    return elm == null ? undefined : {
+      tag: elm.tagName,
+      data: {},
+      children: [].map.call(elm.childNodes, function(child){hydrate(child)})
+    }
+  }
+
+  if (root.hasChildNodes !== undefined && root.hasChildNodes()) {
+    element = root.children[0]
+    node = hydrate(element)
+  }
+
```

Comment:
But even that link to Vue show that they force the user to target the div in that case with id='app'.  I don't like the implicit nature of this default behavior of appending to the end of the existing children and I feel like it will confuse end users that run into that behavior unexpectedly.